catalog/core/catalog-core-commands/pom.xml

```diff
@@ -82,6 +82,12 @@
             <artifactId>gt-cql</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>ddf.catalog.core</groupId>
+            <artifactId>catalog-core-standardframework</artifactId>
+            <version>${project.parent.version}</version>
```

Comment:
Just an FYI more than anything else: http://stackoverflow.com/questions/1981151/warning-on-using-project-parent-version-as-the-version-of-a-module-in-maven-3#1982027 ${project.version} might be better. In our case it is all the same version now so shouldn't matter